docs/api.md

```diff
@@ -75,6 +115,44 @@ Content:
 }
 ```
 
+### <a name="vote-comment"></a>Vote a comment
+
+Upvote or downvote a comment
+
+#### URL
+
+`/lectures/:lecture-id/comment/:comment-id/vote`
```

Comment:
I'm a total REST noob, so please correct me. From what I understood, it is a good practice to use the plural name for both a collection of something and an individual item. What I mean is that (I left lectures out for simplicity) `/comments` will give a list of all comments, while `/comments/:comment-id` will give a specific comment (notice that I've keept the plural form here). This gives a nice "hackability" that will let you remove the ID from an URI and get all the comments (like a tree structure).